modules/ROOT/partials/misc/advcode-shortcuts.adoc

```diff
@@ -1,13 +1,30 @@
-== Advanced Code Editor search and replace keyboard shortcuts
+== Enhanced Code Editor search and replace keyboard shortcuts
 
-Advanced Code Editor provides the following shortcuts within the code dialog.
+Enhanced Code Editor provides the following shortcuts within the code dialog.
 
 [cols=",,",options="header"]
 |===
-|Action |PC |Mac
-|Find |Ctrl+F |Command+F
-|Find next instance |Ctrl+G |Command+G
-|Find previous instance |Shift+Ctrl+G |Shift+Command+G
-|Replace |Ctrl+H |Command+Option+F
-|Replace All |Shift+Ctrl+R |Shift+Command+Option+F
+|Action |PC |Mac 
+|Find |Ctrl+F |Command+F 
+|Find next instance |Ctrl+G |Command+G 
+|Find previous instance |Shift+Ctrl+G |Shift+Command+G 
+|Replace |Ctrl+H |Command+Option+F 
+|Replace All |Shift+Ctrl+R |Shift+Command+Option+F 
+|Open Search Panel |Ctrl+F |Command+F 
+|Find Next |F3 |F3 
+|Find Previous |Shift+F3 |Shift+F3 
+|Close Search Panel |Esc |Esc 
+|Select Selection Matches |Ctrl+Shift+L |Command+Shift+L 
+|Select Next Occurrence |Ctrl+D |Command+D 
+|Select Next Occurrence (Atom) |Ctrl+B |Command+B 
+|Fold Code |Ctrl+Q |Command+Q+Fn
+|Accept Completion |Tab |Tab 
```

Comment:
We should include the indent shortcuts here too.

Reply:
Im waiting for @spocke to confirm what we are doing with this. But we will include the indent and outdent shortcuts here.

Reply:
@kemister85 about this:
Tested on Linux:
`|Find |Ctrl+F |Command+F` -> It works
`|Find next instance |Ctrl+G |Command+G` -> Need to be removed 
`|Find previous instance |Shift+Ctrl+G |Shift+Command+G` -> Need to be removed
`|Replace |Ctrl+H |Command+Option+F` -> It takes the native event so open the History
`|Replace All |Shift+Ctrl+R |Shift+Command+Option+F` -> it takes the native event so it refresh the page
`|Open Search Panel |Ctrl+F |Command+F` > It works
`|Find Next |F3 |F3` -> It works
`|Find Previous |Shift+F3 |Shift+F3` -> It works
`|Close Search Panel |Esc |Esc` -> It works
`|Select Selection Matches |Ctrl+Shift+L |Command+Shift+L` -> It seems not to work but I'm not sure how
`|Select Next Occurrence |Ctrl+D |Command+D` -> It doesn't work
`|Select Next Occurrence (Atom) |Ctrl+B |Command+B` -> It doesn't work
`|Fold Code |Ctrl+Q |Command+Q+Fn` -> this was remove we need to replaced this row with:
```
Ctrl-Shift-[ (Cmd-Alt-[ on macOS): foldCode.
Ctrl-Shift-] (Cmd-Alt-] on macOS): unfoldCode.
Ctrl-Alt-[: foldAll.
Ctrl-Alt-]: unfoldAll.
```
`|Accept Completion |Tab |Tab` -> this should be replaced with `Enter` because `Tab` at the moment also move the focus on the buttons

Reply:
```suggestion
|=== 
| Action | Windows Shortcut | macOS Shortcut
| Find | Ctrl+F | Command+F
| Replace | Ctrl+H | Command+Option+F
| Replace All | Shift+Ctrl+R | Shift+Command+Option+F
| Find Next | F3 | F3
| Find Previous | Shift+F3 | Shift+F3
| Close Search Panel | Esc | Esc
| Fold Code | Ctrl-Shift-[ | Command-Option-[
| Unfold Code | Ctrl-Shift-] | Command-Option-]
| Fold All | Ctrl-Alt-[ | Ctrl-Option-[
| Unfold All | Ctrl-Alt-] | Ctrl-Option-]
| Accept Completion | Enter | Enter
|=== 

```

Reply:
@lorenzo-pomili I have update the new shortcuts with the newly tested ones.